src/bin/collapse-dtrace.rs

```diff
@@ -26,11 +37,23 @@ struct Opt {
     #[structopt(short = "v", long = "verbose", parse(from_occurrences))]
     verbose: usize,
 
-    /// Include offsets
-    #[structopt(long = "includeoffset")]
-    includeoffset: bool,
+    // Options...
+    /// Number of stacks per job sent to threadpool (only used if nthreads > 1)
+    #[structopt(long = "nstacks", raw(default_value = "&NSTACKS"), value_name = "UINT")]
+    nstacks: usize,
+
+    /// Number of threads to use.
+    #[structopt(
```

Comment:
I'm inclined to not even expose this on the CLI. Thoughts?

Reply:
I'll push back a little on this one. It seems like a pretty straight-forward parameter (not confusing to the user like `nstacks` would have been). So why not let the user change it?

Reply:
Okay, fair enough. I feel less inclined to hide `nthreads` than I do `nstacks` :)